include/ttmlir/Conversion/Passes.h

```diff
@@ -20,6 +20,7 @@
 #include "mlir/Dialect/EmitC/IR/EmitC.h"
 
 namespace mlir::tt {
+std::unique_ptr<OperationPass<ModuleOp>> createTTIRGatherPatternMatch();
```

Comment:
You shouldn't need to do this as you have already registered a GEN_PASS_DEF() for this pass. Look here:
https://github.com/tenstorrent/tt-mlir/compare/main...uaziz/relu-replacement